jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -134,12 +134,15 @@
                  && (bg_selected === 'None' || bg_selected_validity.is_aperture)
                  && dev_subpixel_support">
         <v-row>
```

Comment:
```suggestion
        <v-row v-if="aperture_selected !== 'Entire Cube'">
```

We should also include an in-line warning if `!aperture_selected_validity.is_aperture` conflicts with the selection here (for example a composite subset and "subpixel"), disable clicking the extract button, and have checks in the python method that raise an error.

Reply:
As it currently stands, if you have Subset 1 selected and then you make it a composite subset, the option to make a cone wavelength dependent is removed and you see the following message in the UI: "Subset 1 does not support wavelength dependence (cone support): composite subsets are not supported." Is that sufficient or do you mean to cover another case with this?

Reply:
oh right - I forgot I did that 😝  That's probably fine for now, and I'll make a note of this in the final UI/UX review (we might instead want to continue showing the dropdown and have an error so that there is a consistent workflow from the API perspective).

Reply:
I am no good with the vue stuff but now that cylindrical also use the same algorithm as cone, any checks here need updating?

Reply:
I think this should work with #2684.  If anything there needs updating, please open a ticket.